### ISSUE TITLE
given color isn't a valid rgb or rgba color

### ISSUE THREAD

**Issue by kfirzuberi** [christian-bromann/rgb2hex#4]
Yesterday a new version of the module was published and I started getting this error:
image
![image](https://user-images.githubusercontent.com/6490785/41409343-80e02c76-6fde-11e8-9246-f8ec6d127fb9.png)

I'm using http://webdriver.io and according to their [documentation](http://webdriver.io/api/property/getCssProperty.html), when using getCssProperty to get color,
it uses rgb2hex module.
I didn't change anything in my code. is it a Bug? do I need to change something in order to fix it?

**Comment by ianrenauld** [comment]
I tried to fix the issue (see PR), but after talking to Erwin Heitzman, I realized that the CSS Property I was using (and it seems that you are using) was the wrong one. I got a similar string to yours because I requested the CSS property "background".  However, if I ask for "background-color", then I ONLY get the color and none of the extraneous text that follows it, which is then perfectly converted by the latest version of rgb2hex.  I suggest you take another look at the CSS property you are requesting and try to find one that will give you only the color. That should solve your issue.
